apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.sasquatch.activities;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Spinner;
+
+import com.microsoft.appcenter.sasquatch.R;
+import com.microsoft.appcenter.storage.Constants;
+import com.microsoft.appcenter.storage.models.PaginatedDocuments;
+import com.microsoft.appcenter.utils.async.AppCenterConsumer;
+import com.microsoft.appcenter.utils.async.AppCenterFuture;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+
+import static com.microsoft.appcenter.sasquatch.activities.MainActivity.LOG_TAG;
+
+class TestDocument {
+
+    @SuppressWarnings("unused")
+    String test = "ABC";
+}
+
+public class StorageActivity extends AppCompatActivity {
+
+    private Spinner mStorageTypeSpinner;
+
+    private ListView listView;
+
+    private String[] userDocumentList = {"Doc1-User", "Doc2-User", "Doc3-User", "Doc4-User", "Doc5-User", "Doc6-User", "Doc7-User", "Doc8-User", "Doc9-User", "Doc10-User", "Doc11-User", "Doc12-User"};
```

Comment:
We need to manage that dynamically with add remove buttons.

Reply:
Add it in new commit.